cmd/ferretdb/main.go

```diff
@@ -212,9 +214,17 @@ func run() {
 
 	go debug.RunHandler(ctx, cli.DebugAddr, metricsRegisterer, logger.Named("debug"))
 
+	cmdsList := make([]string, len(common.Commands))
+	for k := range common.Commands {
+		cmdsList = append(cmdsList, k)
+	}
+
+	metrics := connmetrics.NewListenerMetrics(cmdsList)
```

Comment:
```suggestion
	metrics := connmetrics.NewListenerMetrics(maps.Keys(common.Commands))
```

https://pkg.go.dev/golang.org/x/exp/maps#Keys

But we also might want to sort them to make results stable